apertium-eng.eng.rlx

```diff
@@ -329,6 +344,33 @@ SELECT V IF (-1 (prn pers)) ;
 SELECT SUB:1 Be IF (1 Ing) (0/* Have) ;
 SELECT Ger IF (-1/* Be) (1 To) ;
 
+SELECT Adv IF (0 Pr) (-1 Vblex) ;
+#Example (on):
+#He talked on.
+
+SELECT Cnjadv IF (0 Pr) (1 Det) (2 N) ;
+#Example (for):
+#Please get fish for the cat.
+
+SELECT Adv IF (0 N) (1 Supadj) ;
+#Example (smallest)
```

Comment:
smallest?

Reply:
Oh, that's mistake.

Reply:
?

Reply:
Sorry, I meant to type `second` instead of smallest.  I'm not quite sure how that happened.